### PR TITLE
Fix a regexp bug

### PR DESCRIPTION
**Pull request by wangkuiyi**
Fix https://github.com/gangliao/bazel.cmake/issues/20

### CODE REVIEW

**Comment by gangliao** [comment]
That's awesome!!

File: cmake/bazel.cmake [hunk 1]
```diff
@@ -52,7 +52,7 @@ add_custom_target(helps COMMAND ./make_help WORKING_DIRECTORY ${BAZEL_SOURCE_DIR
 
 macro(_build_target func_tag)
   set(_sources ${ARGN})
-  list(FILTER _sources EXCLUDE REGEX ".proto$")
+  list(FILTER _sources EXCLUDE REGEX "\\.proto$")
```
Comment (gangliao) [inline cmake/bazel.cmake]: Not sure what's the meaning of `\\`.

It that similar to `[a-zA-z0-9]+[.]proto$`?

Comment (gangliao) [reply cmake/bazel.cmake]: Regular Expressions http://www.mpi.nl/corpus/html/trova/ch01s04.html

Comment (wangkuiyi) [reply cmake/bazel.cmake]: https://cmake.org/pipermail/cmake/2004-May/005136.html

**Comment by wangkuiyi** [comment]
https://cmake.org/pipermail/cmake/2004-May/005136.html

**Pull request merged**